Assets/Scripts/Player/Movement/PlayerMovement.cs

```diff
@@ -4,18 +4,32 @@ public sealed class PlayerMovement : MonoBehaviour
 {
     [SerializeField] private CharacterController characterController;
     [SerializeField] private float movementSpeed;
+    [SerializeField] private float gravity;
+    [SerializeField] private float groundDistance;
+    [SerializeField] private Transform groundCheck;
+    [SerializeField] private LayerMask groundMask;
```

Comment:
2 keer feedback!!!

Kan je headers toevoegen, want het wordt een beetje veel hierzo. Een header voor refrences en voor settings.

```suggestion
    [SerializeField] private CharacterController characterController;
    [SerializeField] private float movementSpeed;
    [SerializeField, Range(0, 10)] private float gravity;
    [SerializeField, Range(0, 10)] private float groundDistance;
    [SerializeField] private Transform groundCheck;
    [SerializeField] private LayerMask groundMask;
```

Sliders :D